validator/client/propose_protect.go

```diff
@@ -3,20 +3,45 @@ package client
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	"github.com/pkg/errors"
 	fieldparams "github.com/prysmaticlabs/prysm/v4/config/fieldparams"
 	"github.com/prysmaticlabs/prysm/v4/consensus-types/interfaces"
+	"github.com/prysmaticlabs/prysm/v4/validator/db/filesystem"
+	"github.com/prysmaticlabs/prysm/v4/validator/db/kv"
 	"github.com/sirupsen/logrus"
 )
 
-var failedBlockSignLocalErr = "attempted to sign a double proposal, block rejected by local protection"
+var failedBlockSignLocalErr = "block rejected by local protection"
 
 // slashableProposalCheck checks if a block proposal is slashable by comparing it with the
 // block proposals history for the given public key in our DB. If it is not, we then update the history
 // with new values and save it to the database.
 func (v *validator) slashableProposalCheck(
-	ctx context.Context, pubKey [fieldparams.BLSPubkeyLength]byte, signedBlock interfaces.ReadOnlySignedBeaconBlock, signingRoot [32]byte,
+	ctx context.Context,
+	pubKey [fieldparams.BLSPubkeyLength]byte,
+	signedBlock interfaces.ReadOnlySignedBeaconBlock,
+	signingRoot [32]byte,
```

Comment:
you can also use fieldparams.RootLength

Reply:
Fixed for this occurrence and all other `signingRoot` occurrences as well.